web-marketplace/src/components/organisms/RegistryLayout/RegistryLayout.Header.tsx

```diff
@@ -53,6 +53,10 @@ const RegistryLayoutHeader: React.FC = () => {
   const { showProjects, showCreditClasses, isIssuer } = useProfileItems({});
   const menuItems = useMemo(() => getMenuItems(pathname, _), [pathname, _]);
   const onProfileClick = useOnProfileClick();
+
+  // TODO: implement savedPaymentInfo
+  const savedPaymentInfo = true;
```

Comment:
FYI we have already everything in place to query for that, as it's done in https://github.com/regen-network/regen-web/pull/2477/files#diff-6e2329b0dce1124fe41992d4c6ab2fd4e42d13f9c52a0e9b11b6f42148db5556
so it'd be nice to use it

Reply:
Updated accordingly, thanks!